# Удалите образ my_python_app, который был создан ранее.


# Проверьте, что образ больше не отображается в списке локальных образов
